bindings/java/pom.xml

```diff
@@ -58,56 +62,53 @@
         <dependency>
             <groupId>org.questdb</groupId>
             <artifactId>jar-jni</artifactId>
-            <version>${rust-maven-plugin.version}</version>
+            <version>${questdb.version}</version>
         </dependency>
+
         <dependency>
             <groupId>io.cucumber</groupId>
             <artifactId>cucumber-java</artifactId>
             <scope>test</scope>
         </dependency>
-
         <dependency>
             <groupId>io.cucumber</groupId>
             <artifactId>cucumber-junit-platform-engine</artifactId>
             <scope>test</scope>
         </dependency>
-
         <dependency>
             <groupId>org.junit.platform</groupId>
             <artifactId>junit-platform-suite</artifactId>
             <scope>test</scope>
         </dependency>
-
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
-
-
     </dependencies>
 
     <build>
+        <testResources>
```

Comment:
Always follow symbol link, so that `bindings.feature` always work instead of suffering from relative path.